### PR TITLE
Adding SimpleLighting sample. Updates to Hello World sample

### PR DESCRIPTION
**Pull request by pkristof**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
 ## API Samples
 In the Samples directory, you will find samples that attempt to break off specific features and specific usage scenarios into bite sized chunks. For example, the ExecuteIndirect sample will show you just enough about execute indirect to get started with that feature without diving too deep into multiengine whereas the nBodyGravity sample will delve into multiengine without touching on the execute indirect feature etc. By doing this, we hope to make it easier to get started with DirectX 12.
 
 Recent API Sample Updates:
-1. D3D12 Depth Bounds Test: This sample demos how to use depth bounds test with DirectX 12. In addition, it introduces a more flexible way to create pipeline state, called Pipeline State Object Stream (PSO Stream). PSO Stream is required to use Depth Bounds Test feature.
-2. Shader Model 6 Wave Intrinsics: This sample visualizes several new wave intrinsics in Shader Model 6.
+1. [D3D12 Raytracing](Samples\Desktop\D3D12Raytracing\readme.md): This is a set of tutorial and advanced samples that demonstrate the use of DirectX Raytracing.
```
Comment (wallisc) [inline README.md]: tutorials*

File: Samples/Desktop/D3D12Raytracing/readme.md [hunk 2]
```diff
@@ -1,16 +1,38 @@
-
 # D3D12 Raytracing Samples
-This collection of samples act as an introduction to Direct3D 12 Raytracing. Each sample introduces a few new concepts. 
+This collection of samples act as an introduction to DirectX Raytracing (DXR). The samples are divided into tutorials and advanced samples. Each tutorial sample introduces a few new concepts. Advanced samples demonstrate more complex techniques and applications of raytracing. 
+
+The samples implement both DXR and D3D12 Raytracing Fallback Layer APIs. This is purely for demonstration purposes to show where the API differences are. Real-world applications will implement only one or the other. The Fallback Layer uses DXR if a driver and OS supports it. Otherwise, it falls back to compute pipeline to emulate raytracing. Developers aiming for a wider HW support should target the Fallback Layer.
```
Comment (wallisc) [inline Samples/Desktop/D3D12Raytracing/readme.md]: nit: I'd condense "to show where API difference are" -> "to show API differences"

Comment (wallisc) [reply Samples/Desktop/D3D12Raytracing/readme.md]: "it falls back to compute pipeline" -> "it falls back to the compute pipeline"

Comment (wallisc) [reply Samples/Desktop/D3D12Raytracing/readme.md]: "for a wider HW support" -> "for wider HW support"

File: Samples/Desktop/D3D12Raytracing/readme.md [hunk 3]
```diff
@@ -1,16 +1,38 @@
...
 This sample demonstrates integration of the Fallback Layer in the MiniEngine's Model Viewer and several sample uses of raytracing.
 
 ![D3D12 Raytracing Mini Engine](src/D3D12RaytracingMiniEngineSample/Screenshot_small.png)
 
+## Requirements
+* [Visual Studio 2017](https://www.visualstudio.com/) with the [Windows 10 Fall Creators Update SDK](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk)
+* Windows 10 with the Fall Creators Update
+* A graphics card with retail DXIL capabilities.
+* DXIL.dll should be pulled from the latest Windows SDK to accompany the compiler or enable Developer mode.
+* Download DirectXRaytracingBinariesV1.0.zip from https://github.com/Microsoft/DirectX-Graphics-Samples/releases and copy all the contents to Samples/Desktop/D3D12Raytracing/tools/x64
```
[the remark this answers is not in the document]
Comment (pkristof) [reply Samples/Desktop/D3D12Raytracing/readme.md]: Yes, that's why it was put in the common readme and all samples point to it.

**Pull request merged**